Server/mods/deathmatch/logic/luadefs/CLuaGenericDefs.cpp

```diff
@@ -26,3 +38,153 @@ bool CLuaGenericDefs::IsTransferBoxVisible()
 {
     return g_pGame->IsClientTransferBoxVisible();
 }
+
+bool CLuaGenericDefs::ClearChatBox(std::optional<CElement*> element)
+{
+    CStaticFunctionDefinitions::ClearChatBox(element.value_or(m_pRootElement));
+    return true;
+}
+
+std::variant<bool, std::string, CLuaArguments> CLuaGenericDefs::GetServerConfigSetting(std::string name)
+{
+    // Try as single setting
+    if (SString value; g_pGame->GetConfig()->GetSetting(name, value))
+    {
+        return value;
+    }
+
+    // Try as multiple setting
+    if (CLuaArguments result; g_pGame->GetConfig()->GetSettingTable(name, &result))
+    {
+        return result;
+    }
+
+    return false;
+}
+
+bool CLuaGenericDefs::SetServerPassword(std::optional<std::string> rawPassword)
+{
+    std::string password = (rawPassword.has_value() ? std::move(rawPassword.value()) : std::string());
+
+    if (!CStaticFunctionDefinitions::SetServerPassword(password, true))
+        throw std::invalid_argument("password must be shorter than 32 chars and just contain visible characters");
+
+    return true;
+}
+
+std::variant<std::nullptr_t, std::string_view> CLuaGenericDefs::GetServerPassword()
+{
+    if (g_pGame->GetConfig()->HasPassword())
+    {
+        return std::string_view{g_pGame->GetConfig()->GetPassword()};
+    }
+
+    return nullptr;
+}
+
+unsigned int CLuaGenericDefs::GetServerHttpPort()
+{
+    return g_pGame->GetConfig()->GetHTTPPort();
+}
+
+std::string_view CLuaGenericDefs::GetServerName()
+{
+    return std::string_view{g_pGame->GetConfig()->GetServerName()};
```

Comment:
`return g_pGame->GetConfig()->GetServerName()`

Reply:
```suggestion
    return g_pGame->GetConfig()->GetServerName();
```
Converted to a suggestion, so I can apply it directly.